tools/percolator/rollout_nested_collection.xml

```diff
@@ -0,0 +1,52 @@
+<tool id="rollout_nested_collection" name="Rollout nested list" version="3.1">
+    <description>into a single long list</description>
```

Comment:
Yeah... I think this is what flatten does - or pretty close https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/tools/flatten_collection.xml. Can you try to use flatten and if it isn't quite right clarify how and maybe we can options to make it work the way you are hoping to.